src/mame/audio/nl_astrob.cpp

```diff
@@ -464,6 +488,17 @@ NETLIST_START(astrob)
 	Q_2N4403(Q10)
 
 	TL084_DIP(U1)           // Op. Amp.
+#if (ENABLE_SONAR_ALT)
+	// Oscillators are of order 1kHz. No need to to have UGF in MHz range
+	PARAM(U1.A.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U1.B.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U1.C.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U1.D.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U2.A.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U2.B.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U2.C.MODEL, "TL084(TYPE=3 UGF=10k)")
+	PARAM(U2.D.MODEL, "TL084(TYPE=3 UGF=10k)")
+#endif
```

Comment:
UGF = Unity Gain Frequency, usually identical to gain-bandwidth product for op-amps. The idea is that the an op-amp's open-loop gain (its amplification factor when used without any negative feedback) is proportional to the frequency of its input signal. For slowly-varying, near-DC signals, this gain will be extremely high (in this case, a gain of 10000 at 1 Hz), which is part of what makes an op-amp effective. In practice the op-amp's actual gain will be limited by whatever negative feedback is applied. Because the op-amp is limited in how fast it can respond to its input, its open-loop gain will drop as the frequency of the input signal rises, eventually falling to a gain of 1 (output = input) at the unity gain frequency. In this case couriersud set the unity gain frequency is 10 kHz, which is much lower than reality, in order to make the op-amp response in the oscillators slower and easier for the netlist solver to handle without using very small timesteps. According to Texas Instruments, the TL084's actual gain-bandwidth product (remember, this is the same as the UGF) is typically 3 MHz.